File: Trabajos-Intr a la Ing de Sistem/Trabajos python/Ejercicio 1.py

#PROMEDIO DE NOTAS:
print("PROMEDIO DE NOTAS:")

cantidad = int(input("¿Cuántas notas desea ingresar?: "))

suma = 0

for i in range(cantidad):
    nota = float(input(f"Ingrese la nota {i + 1}: "))
    suma += nota

promedio = suma / cantidad

print(f"\nEl promedio final es: {promedio:.2f}")

if promedio >= 11:
    print("Estado: APROBADO")
else:
    print("Estado: DESAPROBADO")

File: Trabajos-Intr a la Ing de Sistem/Trabajos C#/Ejercicio 1.cs
using System;

class Programat1
{
    static void Main()
    {
        // PROMEDIO DE NOTAS:
        Console.WriteLine("PROMEDIO DE NOTAS:");

        Console.Write("¿Cuántas notas desea ingresar?: ");
        int cantidad = int.Parse(Console.ReadLine() ?? "0");

        double suma = 0;

        // Bucle para ingresar las notas
        for (int i = 0; i < cantidad; i++)
        {
            Console.Write($"Ingrese la nota {i + 1}: ");
            double nota = double.Parse(Console.ReadLine() ?? "0");
            suma += nota;
        }

        // Calcular el promedio
        double promedio = suma / cantidad;

        // Mostrar el promedio final con 2 decimales
        Console.WriteLine($"\nEl promedio final es: {promedio:F2}");

        // Verificar el estado del alumno
        if (promedio >= 11)
        {
            Console.WriteLine("Estado: APROBADO");
        }
        else
        {
            Console.WriteLine("Estado: DESAPROBADO");
        }
    }
}
